### PR TITLE
Implement vertical resizing

### PR DESCRIPTION
**Pull request by NiGhTTraX**
resize

### CODE REVIEW

**Comment by NiGhTTraX** [comment]
@skidding please review.

File: spec/findPositionForItemSpec.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      var grid;
+
+      beforeEach(function() {
+        grid = new GridList(
+            [{x: 0, y: 0, w: 1, h: 1}, {x: 0, y: 1, w: 1, h: 1}],
+            {rows: 2});
+      });
+
+      it("should place it on a new column on the row I tell it to", function() {
+        expect(grid.findPositionForItem(item, {x: 0, y: 0}, 1)).toEqual([1, 1]);
```
Comment (skidding) [inline spec/findPositionForItemSpec.js]: How come you don't also have a test that checks that the item will be positioned at `[1, 1]` even without passing the `fixedRow` argument...

**Comment by skidding** [comment]
LGTM!

**Comment by skidding** [comment]
:hatched_chick:

File: spec/findPositionForItemSpec.js (hunk 1 above)
Comment (NiGhTTraX) [inline spec/findPositionForItemSpec.js]: It won't be positioned at `[1, 1]` if you don't force the row or the start position. This test is meant to check whether the fixed row is respected when the item is placed at the end of the grid.

Comment (skidding) [inline spec/findPositionForItemSpec.js]: Got it.

**Pull request merged**